doc/sphinx/practical-tools/utilities.rst

```diff
@@ -262,6 +268,40 @@ The following makefile rules can be extended.
   One can append lines to the generated ``.merlin`` file extending this
   target.
 
+.. _coqmakefilelocallate:
+
+CoqMakefile.local-late
+++++++++++++++++++++++
+
+The optional file ``CoqMakefile.local-late`` is included at the end of the generated
+file ``CoqMakefile``.  The following is an incomplete list of accessible variables:
+
+:COQ_VERSION:
+   describes the version of ``coqc`` being used, and can be used to
+   provide different behavior depending on what version of Coq is
+   being used
+:COQMAKEFILE_VERSION:
+   describes the version of Coq that was used to generate the
+   Makefile, and can be used to detect version mismatches
```

Comment:
```suggestion
   the version of Coq that used to generate the
   Makefile, which can be used to detect version mismatches
```

Reply:
You forgot to delete `that` before `was`, but thanks!
```suggestion
   the version of Coq used to generate the
   Makefile, which can be used to detect version mismatches
```